projects/addon-commerce/utils/get-currency-symbol.ts

```diff
@@ -1,7 +1,9 @@
 import {TuiCurrency, TuiCurrencyCode} from '@taiga-ui/addon-commerce/enums';
 import {TuiCurrencyVariants} from '@taiga-ui/addon-commerce/types';
 
-export function tuiGetCurrencySymbol(currency: TuiCurrencyVariants): string | null {
+export function tuiGetCurrencySymbol(
+    currency?: TuiCurrencyVariants | null,
```

Comment:
I have doubts here. Do we really need it ?

Reply:
fixed